src/modules/opengl.c

```diff
@@ -372,6 +452,11 @@ static const char* glPrint(FFinstance* instance)
             error = glxPrint(instance);
     #endif
 
+    #ifdef FF_HAVE_SDL2
+        if(error != NULL)
+            error = sdl2Print(instance);
+    #endif
+
```

Comment:
SDL will always use CGL on MacOS, so i don't see a need to go through its abstraction.